src/main/java/be/seeseemelk/mockbukkit/configuration/GlobalServerConfiguration.java

```diff
@@ -0,0 +1,210 @@
+package be.seeseemelk.mockbukkit.configuration;
+
+import net.kyori.adventure.text.Component;
+import org.jetbrains.annotations.NotNull;
+
+public class GlobalServerConfiguration
```

Comment:
Do all these setters need to be duplicated in the `ServerMock` class? Perhaps we could add to the getters' Javadocs pointing to these setters.

Reply:
Discussed on Discord but decided this is fine.